qa/L0_backend_python/test.sh

```diff
@@ -298,42 +308,47 @@ done
 kill $SERVER_PID
 wait $SERVER_PID
 
-(cd env && bash -ex test.sh)
-if [ $? -ne 0 ]; then
-  RET=1
-fi
+# Disable env test for Jetson since build is non-dockerized and cloud storage repos are not supported
```

Comment:
One comment that I forgot, please file a ticket for enabling a subset of the env tests in Jetson since it touches on various platform specific settings and it might not work on jetson. It is a widely used feature in Python backend and I think we should have some testing for it.

Reply:
Sure. We will ideally need to wait for a dockerized jetson build because we do not want to play with the environments on the CI machine. Additionally the AWS cloud storage support is not added on jetson yet so they will remain disabled.

Reply:
PS: Filed the ticket - DLIS 3377

Reply:
Thanks Hemant.